content/maxwell3_fdem/MT/MT_N_layered_Earth.rst

```diff
@@ -0,0 +1,187 @@
+.. _MT_N_layered_Earth:
+
+Response of a layered Earth to a plane wave
+===========================================
+
+This work follows the derivation in :cite:`ward1988` and is supported by interactive apps developed in a `binder`_.
+
+.. _binder: http://mybinder.org/repo/ubcgif/em_examples/notebooks/geophysical_surveys/MT_N_Layered_Earth/MT_n_layered_earth_example.ipynb
+
+
+
+
+Introduction
+------------
+
+ .. figure:: images/MT_N_layered_Earth-1.hires.png
+    :align: center
+    :scale: 20% 
+    :name: MTlayeredEarth
+
+    Earth Model Configuration
+
+The problem setup is shown in the figure below (See :numref:`MTlayeredEarth`), where we have
+
+| -a layered Earth, each layer with its own physical properties \\(\\sigma, \\varepsilon, \\mu \\) 
+
+| -a plane wave coming from air, perpendicular to the Earth's surface, traveling along the axis Z (oriented downward), with an electric field \\(\\mathbf{E_x}\\) and a magnetic field \\(\\mathbf{H_y}\\)
+
+| - the origin of coordinate system coincides with the Earth's surface
+
+
+
+
+
+
+Governing Equations
+-------------------
+
+The governing equation for DC resistivity problem can be obtained from
+:ref:`Maxwell's equations <maxwell1_fundamentals_index>`. We start by
+considering the zero-frequency case, in which case, Maxwell's equations are
+
+.. math::
+	\nabla \times \mathbf{E_x} = - i \omega \mu \mathbf{H_y}
+	:label: Faraday
+	
+.. math::
+	\nabla \times \mathbf{H_y} = (\sigma + i \omega \varepsilon) \mathbf{E_x}
+    :label: Ampere
+
+Knowing that the divergent of **E** and **H** are equal to 0 here (no free charge)
+according to :eq:`gauss_electric_frequency` and :eq: `gauss_magnetic_frequency`, we can combine the equations to write the Helmhotz (wave propagation) equation for both **E** and **B** field:
```

Comment:
I don't think equations reference properly across pages, so it may be better to link the pages instead